designs/administrator/olm/catalogsource-as-providertype/index.md

```diff
@@ -0,0 +1,24 @@
+---
+parent: Administrator
+version: 4.5
+---
+
+# Custom CatalogSource as OperatorHub ‘Provider Type’ filter facet
+
+When a custom CatalogSource is added today, it appears in the “Custom” facet in the ‘Provider Type’ filter in OperatorHub. Customers are interested in having these CatalogSources show up as their own facet instead of being lumped under “Custom.”
+
+A new annotation will be added to CatalogSources that allows the creator to specify a display-name of that CatalogSource. This value will appear as the facet in ‘Provider Type’ filter. If a display-name isn’t specified, the actual CatalogSource Name will be used.
+
+The "Custom" provider type will no longer exist going forward, and is instead replaced by each CatalogSource being surfaced uniquely. 
+
+## OperatorHub ‘Provider Type’ filter
+
+![OperatorHub Provider Type filter](img/1-1-OH.png)
```

Comment:
Should all CatalogSource names below "Marketplace" be alphabetized? So in this screen `Our Operators` and `console3-test-operat...` would switch positions?

And I have no idea how many custom CatalogSources is normal, but should we suggest a point at which a "Show # more" button should appear, similar to what we do for the Provider facet?

![image](https://user-images.githubusercontent.com/9122899/76896883-11efbb80-6869-11ea-94fd-0f8a0426ff19.png)

Reply:
Good points Andy, @robszumski do you have any thoughts on always alphabetizing all Provider Types from now on? Or are 'Red Hat' 'Certified' 'Community' expected to be up top? Also, do we always need to show all of them or could there be a breakpoint where they collapse into the "Show # more" button? Will there generally not be that many Provider Types?

![Screen Shot 2020-03-18 at 4 17 14 PM](https://user-images.githubusercontent.com/54860125/77003658-23ea6080-6934-11ea-866a-d068425ab308.png)

Reply:
Good questions.

> do you have any thoughts on always alphabetizing all Provider Types from now on? Or are 'Red Hat' 'Certified' 'Community' expected to be up top? 

If that was easy, I don't see why not to do it. But it's not required IMO.

> Also, do we always need to show all of them or could there be a breakpoint where they collapse into the "Show # more" button? Will there generally not be that many Provider Types?

I wouldn't anticipate there being too many of these, but if we picked up this behavior for free due to how the other filters work, that seems ok to me. 5-6 total catalogs would be the upper bound of most users.

Reply:
Added these changes, thanks.